pages/lists/addplanttoplace.js

```diff
@@ -36,25 +39,31 @@ export default function AddPlantToPlaceList({ places, setPlaces }) {
       <>
         {plantsSorted.map((plant) => {
           return (
-            <section key={plant.id}>
+            <React.Fragment key={plant.id}>
```

Comment:
This works but is not really needed. You can just add the `key` to `StyledCard`:

```jsx
<StyledCard border={"green"} key={plant.id}>
```

then you can get rid of the Fragment alltogether.

Reply:
Oh, I tried, but kept getting iene errors in Firefox. Therefore the way with the fragment. I try it again.